src/app/index.js

```diff
@@ -15,6 +16,13 @@ class Server {
     this.app.get('/', basicAuth.isAuth, (req, res) => {
       res.send('This service is running properly ')
     })
+    this.app.get(
+      '/api/getListDevice',
```

Comment:
kalo rest biasanya cukup `api/device` untuk list dan `api/device/:id` kalo mau dapet detailnya

dan kalo mau add cukup `/api/device`atau bisa juga `api/device/add` trus kan udah keliatan nih pasti pake `api/device` nah itu bisa pake `this.app.use` aja buat main route

Reply:
karena endpoint dikit saya gak pake route mas ,
jadi gapapa di jadiin 1?

Reply:
boleh